lib/galaxy/jobs/runners/cli.py

```diff
@@ -50,8 +52,9 @@ def parse_destination_params( self, params ):
     def queue_job( self, job_wrapper ):
         """Create job script and submit it to the DRM"""
         # prepare the job
-        if not self.prepare_job( job_wrapper, include_metadata=True ):
-            return
+        include_metadata = asbool( job_wrapper.job_destination.params.get( "embed_metadata_in_job", DEFAULT_EMBED_METADATA_IN_JOB ) )
+        if not self.prepare_job( job_wrapper, include_metadata=include_metadata):
+             return
```

Comment:
Here `return` is indented with 5 chars with respect to the line above, should be 4.